scripts/base/frameworks/files/main.zeek

```diff
@@ -335,7 +335,7 @@ global registered_protocols: table[Analyzer::Tag] of ProtoRegistration = table()
 
 # Store the MIME type to analyzer mappings.
 global mime_types: table[Files::Tag] of set[string];
-global mime_type_to_analyzers: table[string] of set[Files::Tag];
+global mime_type_to_analyzers: table[string] of vector of Files::Tag;
```

Comment:
This may be a matter of taste, but I'm wondering if we should leave this a `set`, and instead just convert into a sorted vector on output (i.e., when adding it to the logged `analyzers` field).